orttraining/orttraining/python/ort_trainer.py

```diff
@@ -845,6 +870,56 @@ def _prepare_input_and_fetches(self, input_desc_with_, internal_learning_rate, i
 
         return input, fetches
 
+    def _debug_conversion_step(self, input, input_descs, output_desc):
+        # handle case if onnx model is provided instead of torch
```

Comment:
Maybe we can add an informational message here, something like 'Comparing inference output between PyTorch and ONNX models'